src/ast/file_splitter.rs

```diff
@@ -73,8 +73,8 @@ impl AstBasedFileSplitter {
             }
         };
 
-        let mut files_markup: HashMap<String, Arc<crate::scratchpads::chat_utils_rag::File>> = HashMap::new();
-        files_markup.insert(path_str.to_string(), Arc::new(crate::scratchpads::chat_utils_rag::File { markup: ast_markup.clone(), cpath: path.clone(), cpath_symmetry_breaker: 0.0 }));
+        // let mut files_markup = HashMap::new();
```

Comment:
why it's commented?

Reply:
files_markup isn't used anywhere from what I can gather

Reply:
Then why did you change it?

Reply:
it shouldn't be there -- as it's a dead code -- it does nothing

Reply:
Just get rid of the dead code then
Or do not touch it

Reply:
You're making an impression that the file splitter is still alive

Reply:
I have no idea whether you use that file_splitter or not :)
 it's just `files_markup ` is not used there)

Reply:
Then remove those line completely 
Why you just commented them?